webapp/custom-webpack.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
       // https://github.com/enketo/enketo-core/issues/454
       '../../js/dropdown.jquery': 'node_modules/bootstrap/js/dropdown',
       'bikram-sambat': 'node_modules/bikram-sambat',
+      'messageformat': 'node_modules/messageformat/index',
```

Comment:
Do we need to do something else to integrate this one?

Reply:
This should be enough.